src/pgn.ts

```diff
@@ -445,7 +445,7 @@ export class PgnParser {
 
   private handleLine() {
     let freshLine = true;
-    let line = this.lineBuf.join('');
+    let line = this.lineBuf.join('').replace(/^\s+/, '');
```

Comment:
This looks like it will break comments.

Reply:
Do you have an example for this?

I'll add it as a test, I think there should be one here.